site2/docs/tiered-storage-aliyun.md

```diff
@@ -0,0 +1,242 @@
+---
+id: tiered-storage-aliyun
+title: Use Aliyun OSS offloader with Pulsar
+sidebar_label: Aliyun OSS offloader
+---
+
+This chapter guides you through every step of installing and configuring the Aliyun Object Storage Service (OSS) offloader and using it with Pulsar.
+
+## Installation
+
+Follow the steps below to install the Aliyun OSS offloader.
+
+### Prerequisite
+
+- Pulsar: 2.8.0 or later versions
+  
+### Step
+
+This example uses Pulsar 2.8.0.
+
+1. Download the Pulsar tarball using one of the following ways:
+
+   * Download from the [Apache mirror](https://archive.apache.org/dist/pulsar/pulsar-2.8.0/apache-pulsar-2.8.0-bin.tar.gz)
+
+   * Download from the Pulsar [downloads page](https://pulsar.apache.org/download)
+
+   * Use [wget](https://www.gnu.org/software/wget):
+
+     ```shell
+     wget https://archive.apache.org/dist/pulsar/pulsar-2.8.0/apache-pulsar-2.8.0-bin.tar.gz
+     ```
```

Comment:
Those info duplicate with info here https://pulsar.apache.org/docs/en/standalone/#install-pulsar-using-binary-release, we can use reference to that link directly.

Reply:
Fixed